src/repositories/hive/hive.repository.mock.ts

```diff
@@ -75,4 +75,8 @@ export class MockHiveRepository extends HiveRepository {
       expired: false,
     };
   }
+
+  verifyHiveMessage(message: Buffer, signature: string, account: ExtendedAccount): boolean {
```

Comment:
I think this should be the real implementation, just so integration tests can function normally.

Reply:
If we want the real implementation we've gotta use real private keys to sign the message orrr mock the getAccount feature, injecting our own public keys so it always passes but that kinda defeats the purpose and we might as well just have it return true.

Reply:
resolved